apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/CommonCodings.java

```diff
@@ -0,0 +1,27 @@
+package gov.cms.bfd.server.war.commons;
+
+public enum CommonCodings {
```

Comment:
Please add documentation on _what_ codings these are for, and what belongs in this class. Looking at this with little domain knowledge, I wouldnt know when to add a new value here or what it's used for.

Reply:
I also have little domain knowledge, I don't know anything beyond the fact that I had to use it in more than one place, so I made a class to stash it in.  If we have a suitable way to classify this, or a better existing place for it, I'm open to suggestions.

Reply:
Moved this to the `IdentifierType` enum class as suggested by BFD.